include/cudaq/Optimizer/CallGraphFix.h

```diff
@@ -0,0 +1,47 @@
+/****************************************************************-*- C++ -*-****
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#pragma once
+
+#include "mlir/Analysis/CallGraph.h"
+
+namespace llvm {
+// FIXME: `GraphTraits` specialization for `const mlir::CallGraphNode *` in
```

Comment:
MLIR eschews the use of `const`. https://mlir.llvm.org/docs/Rationale/UsageOfConst/